src/zarpaint/reader.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import annotations
-
+import zarr
 import os
 import pathlib
 from ._zarpaint import open_tensorstore, open_ts_meta
```

Comment:
```suggestion
from ._zarpaint import  open_ts_meta
```

We probably no longer need that import? And should also be able to get rid of the function entirely from `._zarpaint`.